subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/ReportResolver.kt

```diff
@@ -3,34 +3,35 @@ package com.avito.instrumentation.internal
 import com.avito.instrumentation.configuration.InstrumentationTestsPluginExtension
 import com.avito.instrumentation.configuration.ReportViewer
 
-internal class ReportResolver(private val runIdResolver: RunIdResolver) {
+internal class ReportResolver(
+    private val extension: InstrumentationTestsPluginExtension,
+    private val runIdResolver: RunIdResolver
+) {
 
-    fun getReportApiUrl(extension: InstrumentationTestsPluginExtension): String {
+    fun getReportApiUrl(): String {
```

Comment:
```suggestion
    val reportApiUrl: String
```
Why we don't use the property?

Reply:
i think we can

Reply:
ReportViewer model is not a properties, just var's https://github.com/avito-tech/avito-android/blob/71c71f8eb633b4e4bfc81984fe3f62b967b3806a/subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationTestsReportExtension.kt#L21

so it should be queried lazily, properties with getters will work, but not with eager initializers